readme.md

```diff
@@ -25,6 +25,21 @@ Add `@ava/stage-4` to your [Babel] presets.
 Require `@ava/babel-preset-stage-4/package-hash` to get a combined hash for the installed version of the preset, as well as the plugins that were selected for the active Node.js version.
 
 
+## Options
+
+For more information on setting options for a preset, refer to the [plugin/preset options] documentation.
+
+
+### `modules`
+
+`"commonjs" | false`, defaults to `"commonjs"`.
+
+Enable transformation of ES6 module syntax to another module type.
```

Comment:
I'd prefer `ES2015` over `ES6`. That also corresponds with the plugin package name.

Reply:
ok, it was taken from env preset.